Scripts/Core/EditorVR.Rays.cs

```diff
@@ -67,6 +67,18 @@ public void ConnectInterface(object obj, Transform rayOrigin = null)
 					if (ray != null)
 						ray.rayOrigin = rayOrigin;
 
+					var rayOrigins = obj as IUsesRayOrigins;
+					if (rayOrigins != null)
```

Comment:
IForEachRayOrigin is an internal interface. I also wasn't really the mastermind behind the decision to introduce IUsesRayOrigins so I can't really say this is the best way to do it.  Another approach would be to use ILinkedObjects to get the rayorigin from the other AnnotationTool.

That said, a simple interface to get all of the rayorigins in the system seems useful.

Reply:
Let's do this:
Rays implements IForEachRayOrigin
Rename Rays internal ForEachRayOrigin method to something like "IterateRayOrigins", and then everything else will keep working as expected, as it will now have access to the extension method ForEachRayOrigin